_studio/shared/umc/codec/jpeg_dec/src/jpegdec_base.cpp

```diff
@@ -157,18 +159,22 @@ JERRCODE CJPEGDecoderBase::Clean(void)
 
 #define BS_BUFLEN 16384
 
-JERRCODE CJPEGDecoderBase::SetSource(
-  CBaseStreamInput* pInStream)
+JERRCODE CJPEGDecoderBase::SetSource(const uint8_t* pBuf, size_t buflen)
 {
-  JERRCODE jerr;
+  JERRCODE jerr = m_stream_in.Open(pBuf, buflen);
 
-  jerr = m_BitStreamIn.Attach(pInStream);
   if(JPEG_OK != jerr)
     return jerr;
 
+  m_BitStreamIn.Attach(&m_stream_in);
   return m_BitStreamIn.Init(BS_BUFLEN);
 } // CJPEGDecoderBase::SetSource()
 
+JERRCODE CJPEGDecoderBase::Seek(long offset, int origin)
+{
+  return m_stream_in.Seek(offset, origin);
+}// CJPEGDecoderBase::Seek
```

Comment:
useless comment. We already know what function had just finished. It makes some sense for large functions which occupies several screens, but still not much, modern IDEs/text editors easily find end of scope brackets

Reply:
I just keep local code style
If remove this comment, need remove all another - changes in many lines (which not relate to PR).
But if it really need, I can make separate commit in this PR for remove useless comment in jpegdec_base.h/cpp (actually, what about another files?)

Reply:
Yes, I understand, but this function was introduced by you, so you could not copy bad practice within it. Definitely it is really minor issue, I don't want to waste your time to fix it here/in another files. It is just remark from me :)